app/build.gradle

```diff
@@ -318,6 +318,8 @@ android.applicationVariants.all { variant ->
             buildConfigField 'String', 'ADJUST_TOKEN', '"' + token + '"'
             println "(Added from .adjust_token file)"
         } catch (FileNotFoundException ignored) {
+            // Release builds will crash without a dummy file
+            new File("${rootDir}/.adjust_token").text = "--"
```

Comment:
If I recall correctly, crashing was intentional to make sure we weren't shipping a build without one?

Reply:
I'll re-examine this PR once the repo migration will happen, but I believe this comment was about the actual build process and not the finished artifact. IIRC I had removed the empty `.adjust_token` file as part of this PR but we still wanted it present and empty during pipeline builds.